config/test.exs

```diff
@@ -27,7 +27,7 @@ config :lager,
 config :astarte_vmq_plugin, :amqp_options, host: System.get_env("RABBITMQ_HOST") || "rabbitmq"
 
 # make amqp supervisors logs less verbose
-config :logger, handle_otp_reports: false
+config :logger, handle_otp_reports: true
```

Comment:
This defaults to `true`, and the comment above is not relevant anymore. Remove both lines.